src/importers/jiraCsv/JiraCsvImporter.ts

```diff
@@ -62,15 +183,50 @@ export class JiraCsvImporter implements Importer {
     }
 
     for (const row of data) {
-      const url = this.organizationName
-        ? `https://${this.organizationName}.atlassian.net/browse/${row['Issue key']}`
-        : undefined;
-      const mdDesc = j2m.to_markdown(row['Description']);
+      let mdDesc: string = j2m.to_markdown(row['Description']);
+
+      // add attachment links to description
+      let jiraAttachments = row.Attachment || [];
+      if (jiraAttachments && typeof jiraAttachments === 'string') {
+        jiraAttachments = [jiraAttachments];
+      }
+      if (jiraAttachments.length) {
+        mdDesc += `${mdDesc && '\n\n'}Attachment(s):`;
```

Comment:
Instead of always checking for mdDesc existence, maybe just all `|| ''` to line 186